graph/cpp/Makefile

```
@@ -1,8 +1,9 @@
-.PHONY: all clean
+-include ../../Makefile.defaults
 
-all:
-	./make.sh
+compile:
+	CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cmake -B build --fresh -DGRAPH_RECURSIVE=ON -DGRAPH_ABSL_FLAT=ON
```

```suggestion
	cmake -B build --fresh -DCMAKE_BUILD_TYPE=Release -DGRAPH_RECURSIVE=ON -DGRAPH_ABSL_FLAT=ON
```

`CXX` and `CXXFLAGS` should already be passed down from environment. We should specify a release build type.